### PR TITLE
Reduce test time

### LINKED ISSUE #367: Make Unit Tests Faster
The unit tests take too long to run. Investigate which are slow (a few are taking almost all the time) and see about reducing the problem sizes, etc.

### PR DESCRIPTION
**Pull request by streeve**
Reduce mesh size or particle count, focused on the highest previous test times

Closes #367

### CODE REVIEW

**Comment by sslattery** [comment]
@streeve Any before/after timings?

**Comment by streeve** [comment]
> @streeve Any before/after timings?

Yeah I'll add that; looks like I need to fix the Halo first

**Comment by sslattery** [comment]
> Yeah I'll add that; looks like I need to fix the Halo first

Its possible that you made the problem too small and the halo wraps around and double counts. I have done that before.

**Comment by streeve** [comment]
Comparing  (ubuntu:latest, g++, ON, Debug, 3.2.01, NoArborX, NoheFFTe, NoHYPRE, OFF) 

Old:  `Total Test time (real) = 111.38 sec`
New: `Total Test time (real) =  28.73 sec`

Old
```
Test project /__w/Cabana/Cabana/build
      Start  1: Cabana_Version_test
 1/73 Test  #1: Cabana_Version_test ..........................   Passed    0.01 sec
      Start  2: Cabana_Index_test
 2/73 Test  #2: Cabana_Index_test ............................   Passed    0.01 sec
      Start  3: Cabana_CartesianGrid_test
 3/73 Test  #3: Cabana_CartesianGrid_test ....................   Passed    0.01 sec
      Start  4: Cabana_SoA_test
 4/73 Test  #4: Cabana_SoA_test ..............................   Passed    0.01 sec
      Start  5: Cabana_AoSoA_test_OPENMP_nt_1
 5/73 Test  #5: Cabana_AoSoA_test_OPENMP_nt_1 ................   Passed    0.01 sec
      Start  6: Cabana_AoSoA_test_OPENMP_nt_2
 6/73 Test  #6: Cabana_AoSoA_test_OPENMP_nt_2 ................   Passed    0.01 sec
      Start  7: Cabana_DeepCopy_test_OPENMP_nt_1
 7/73 Test  #7: Cabana_DeepCopy_test_OPENMP_nt_1 .............   Passed    0.04 sec
      Start  8: Cabana_DeepCopy_test_OPENMP_nt_2
 8/73 Test  #8: Cabana_DeepCopy_test_OPENMP_nt_2 .............   Passed    0.04 sec
      Start  9: Cabana_LinkedCellList_test_OPENMP_nt_1
 9/73 Test  #9: Cabana_LinkedCellList_test_OPENMP_nt_1 .......   Passed    0.01 sec
      Start 10: Cabana_LinkedCellList_test_OPENMP_nt_2
10/73 Test #10: Cabana_LinkedCellList_test_OPENMP_nt_2 .......   Passed    0.01 sec
      Start 11: Cabana_NeighborList_test_OPENMP_nt_1
11/73 Test #11: Cabana_NeighborList_test_OPENMP_nt_1 .........   Passed    5.75 sec
      Start 12: Cabana_NeighborList_test_OPENMP_nt_2
12/73 Test #12: Cabana_NeighborList_test_OPENMP_nt_2 .........   Passed    2.91 sec
      Start 13: Cabana_Parallel_test_OPENMP_nt_1
13/73 Test #13: Cabana_Parallel_test_OPENMP_nt_1 .............   Passed    0.01 sec
      Start 14: Cabana_Parallel_test_OPENMP_nt_2
14/73 Test #14: Cabana_Parallel_test_OPENMP_nt_2 .............   Passed    0.01 sec
      Start 15: Cabana_ParameterPack_test_OPENMP_nt_1
15/73 Test #15: Cabana_ParameterPack_test_OPENMP_nt_1 ........   Passed    0.01 sec
      Start 16: Cabana_ParameterPack_test_OPENMP_nt_2
16/73 Test #16: Cabana_ParameterPack_test_OPENMP_nt_2 ........   Passed    0.01 sec
      Start 17: Cabana_Slice_test_OPENMP_nt_1
17/73 Test #17: Cabana_Slice_test_OPENMP_nt_1 ................   Passed    0.01 sec
      Start 18: Cabana_Slice_test_OPENMP_nt_2
18/73 Test #18: Cabana_Slice_test_OPENMP_nt_2 ................   Passed    0.01 sec
      Start 19: Cabana_Sort_test_OPENMP_nt_1
19/73 Test #19: Cabana_Sort_test_OPENMP_nt_1 .................   Passed    0.05 sec
      Start 20: Cabana_Sort_test_OPENMP_nt_2
20/73 Test #20: Cabana_Sort_test_OPENMP_nt_2 .................   Passed    0.06 sec
      Start 21: Cabana_Tuple_test_OPENMP_nt_1
21/73 Test #21: Cabana_Tuple_test_OPENMP_nt_1 ................   Passed    0.01 sec
      Start 22: Cabana_Tuple_test_OPENMP_nt_2
22/73 Test #22: Cabana_Tuple_test_OPENMP_nt_2 ................   Passed    0.01 sec
      Start 23: Cabana_CommunicationPlan_test_OPENMP_np_1
23/73 Test #23: Cabana_CommunicationPlan_test_OPENMP_np_1 ....   Passed    0.41 sec
      Start 24: Cabana_CommunicationPlan_test_OPENMP_np_2
24/73 Test #24: Cabana_CommunicationPlan_test_OPENMP_np_2 ....   Passed    0.33 sec
      Start 25: Cabana_Distributor_test_OPENMP_np_1
25/73 Test #25: Cabana_Distributor_test_OPENMP_np_1 ..........   Passed    0.28 sec
      Start 26: Cabana_Distributor_test_OPENMP_np_2
26/73 Test #26: Cabana_Distributor_test_OPENMP_np_2 ..........   Passed    0.34 sec
      Start 27: Cabana_Halo_test_OPENMP_np_1
27/73 Test #27: Cabana_Halo_test_OPENMP_np_1 .................   Passed    0.32 sec
      Start 28: Cabana_Halo_test_OPENMP_np_2
28/73 Test #28: Cabana_Halo_test_OPENMP_np_2 .................   Passed    0.33 sec
      Start 29: Cajita_GlobalMesh_test_OPENMP_nt_1
29/73 Test #29: Cajita_GlobalMesh_test_OPENMP_nt_1 ...........   Passed    0.01 sec
      Start 30: Cajita_GlobalMesh_test_OPENMP_nt_2
30/73 Test #30: Cajita_GlobalMesh_test_OPENMP_nt_2 ...........   Passed    0.01 sec
      Start 31: Cajita_IndexSpace_test_OPENMP_nt_1
31/73 Test #31: Cajita_IndexSpace_test_OPENMP_nt_1 ...........   Passed    0.02 sec
      Start 32: Cajita_IndexSpace_test_OPENMP_nt_2
32/73 Test #32: Cajita_IndexSpace_test_OPENMP_nt_2 ...........   Passed    0.02 sec
      Start 33: Cajita_SparseIndexSpace_test_OPENMP_nt_1
33/73 Test #33: Cajita_SparseIndexSpace_test_OPENMP_nt_1 .....   Passed   24.85 sec
      Start 34: Cajita_SparseIndexSpace_test_OPENMP_nt_2
34/73 Test #34: Cajita_SparseIndexSpace_test_OPENMP_nt_2 .....   Passed   12.58 sec
      Start 35: Cajita_Splines_test_OPENMP_nt_1
35/73 Test #35: Cajita_Splines_test_OPENMP_nt_1 ..............   Passed    0.01 sec
      Start 36: Cajita_Splines_test_OPENMP_nt_2
36/73 Test #36: Cajita_Splines_test_OPENMP_nt_2 ..............   Passed    0.01 sec
      Start 37: Cajita_GlobalGrid_test_OPENMP_np_1
37/73 Test #37: Cajita_GlobalGrid_test_OPENMP_np_1 ...........   Passed    0.32 sec
      Start 38: Cajita_GlobalGrid_test_OPENMP_np_2
38/73 Test #38: Cajita_GlobalGrid_test_OPENMP_np_2 ...........   Passed    0.33 sec
      Start 39: Cajita_LocalGrid_test_OPENMP_np_1
39/73 Test #39: Cajita_LocalGrid_test_OPENMP_np_1 ............   Passed    0.28 sec
      Start 40: Cajita_LocalGrid_test_OPENMP_np_2
40/73 Test #40: Cajita_LocalGrid_test_OPENMP_np_2 ............   Passed    0.33 sec
      Start 41: Cajita_IndexConversion_test_OPENMP_np_1
41/73 Test #41: Cajita_IndexConversion_test_OPENMP_np_1 ......   Passed    5.83 sec
      Start 42: Cajita_IndexConversion_test_OPENMP_np_2
42/73 Test #42: Cajita_IndexConversion_test_OPENMP_np_2 ......   Passed    3.62 sec
      Start 43: Cajita_LocalMesh3d_test_OPENMP_np_1
43/73 Test #43: Cajita_LocalMesh3d_test_OPENMP_np_1 ..........   Passed    8.49 sec
      Start 44: Cajita_LocalMesh3d_test_OPENMP_np_2
44/73 Test #44: Cajita_LocalMesh3d_test_OPENMP_np_2 ..........   Passed    4.82 sec
      Start 45: Cajita_LocalMesh2d_test_OPENMP_np_1
45/73 Test #45: Cajita_LocalMesh2d_test_OPENMP_np_1 ..........   Passed    0.32 sec
      Start 46: Cajita_LocalMesh2d_test_OPENMP_np_2
46/73 Test #46: Cajita_LocalMesh2d_test_OPENMP_np_2 ..........   Passed    0.37 sec
      Start 47: Cajita_Array3d_test_OPENMP_np_1
47/73 Test #47: Cajita_Array3d_test_OPENMP_np_1 ..............   Passed    4.45 sec
      Start 48: Cajita_Array3d_test_OPENMP_np_2
48/73 Test #48: Cajita_Array3d_test_OPENMP_np_2 ..............   Passed    2.54 sec
      Start 49: Cajita_Array2d_test_OPENMP_np_1
49/73 Test #49: Cajita_Array2d_test_OPENMP_np_1 ..............   Passed    0.37 sec
      Start 50: Cajita_Array2d_test_OPENMP_np_2
50/73 Test #50: Cajita_Array2d_test_OPENMP_np_2 ..............   Passed    0.35 sec
      Start 51: Cajita_Halo3d_test_OPENMP_np_1
51/73 Test #51: Cajita_Halo3d_test_OPENMP_np_1 ...............   Passed   12.61 sec
      Start 52: Cajita_Halo3d_test_OPENMP_np_2
52/73 Test #52: Cajita_Halo3d_test_OPENMP_np_2 ...............   Passed    8.39 sec
      Start 53: Cajita_Halo2d_test_OPENMP_np_1
53/73 Test #53: Cajita_Halo2d_test_OPENMP_np_1 ...............   Passed    0.40 sec
      Start 54: Cajita_Halo2d_test_OPENMP_np_2
54/73 Test #54: Cajita_Halo2d_test_OPENMP_np_2 ...............   Passed    0.38 sec
      Start 55: Cajita_SplineEvaluation3d_test_OPENMP_np_1
55/73 Test #55: Cajita_SplineEvaluation3d_test_OPENMP_np_1 ...   Passed    1.27 sec
      Start 56: Cajita_SplineEvaluation3d_test_OPENMP_np_2
56/73 Test #56: Cajita_SplineEvaluation3d_test_OPENMP_np_2 ...   Passed    0.82 sec
      Start 57: Cajita_SplineEvaluation2d_test_OPENMP_np_1
57/73 Test #57: Cajita_SplineEvaluation2d_test_OPENMP_np_1 ...   Passed    0.34 sec
      Start 58: Cajita_SplineEvaluation2d_test_OPENMP_np_2
58/73 Test #58: Cajita_SplineEvaluation2d_test_OPENMP_np_2 ...   Passed    0.34 sec
      Start 59: Cajita_Interpolation3d_test_OPENMP_np_1
59/73 Test #59: Cajita_Interpolation3d_test_OPENMP_np_1 ......   Passed    1.39 sec
      Start 60: Cajita_Interpolation3d_test_OPENMP_np_2
60/73 Test #60: Cajita_Interpolation3d_test_OPENMP_np_2 ......   Passed    0.94 sec
      Start 61: Cajita_Interpolation2d_test_OPENMP_np_1
61/73 Test #61: Cajita_Interpolation2d_test_OPENMP_np_1 ......   Passed    0.31 sec
      Start 62: Cajita_Interpolation2d_test_OPENMP_np_2
62/73 Test #62: Cajita_Interpolation2d_test_OPENMP_np_2 ......   Passed    0.35 sec
      Start 63: Cajita_BovWriter_test_OPENMP_np_1
63/73 Test #63: Cajita_BovWriter_test_OPENMP_np_1 ............   Passed    0.42 sec
      Start 64: Cajita_BovWriter_test_OPENMP_np_2
64/73 Test #64: Cajita_BovWriter_test_OPENMP_np_2 ............   Passed    0.42 sec
      Start 65: Cajita_Parallel_test_OPENMP_np_1
65/73 Test #65: Cajita_Parallel_test_OPENMP_np_1 .............   Passed    1.20 sec
      Start 66: Cajita_Parallel_test_OPENMP_np_2
66/73 Test #66: Cajita_Parallel_test_OPENMP_np_2 .............   Passed    0.82 sec
      Start 67: Core_tutorial_01
67/73 Test #67: Core_tutorial_01 .............................   Passed    0.01 sec
      Start 68: Core_tutorial_02
68/73 Test #68: Core_tutorial_02 .............................   Passed    0.01 sec
      Start 69: Core_tutorial_03
69/73 Test #69: Core_tutorial_03 .............................   Passed    0.01 sec
      Start 70: Core_tutorial_04_unmanaged
70/73 Test #70: Core_tutorial_04_unmanaged ...................   Passed    0.01 sec
      Start 71: Core_tutorial_05_openmp
71/73 Test #71: Core_tutorial_05_openmp ......................   Passed    0.01 sec
      Start 72: Core_tutorial_10_neighbor
72/73 Test #72: Core_tutorial_10_neighbor ....................   Passed    0.01 sec
      Start 73: Core_tutorial_10_simd
73/73 Test #73: Core_tutorial_10_simd ........................   Passed    0.01 sec
```

New
```
Test project /__w/Cabana/Cabana/build
      Start  1: Cabana_Version_test
 1/73 Test  #1: Cabana_Version_test ..........................   Passed    0.01 sec
      Start  2: Cabana_Index_test
 2/73 Test  #2: Cabana_Index_test ............................   Passed    0.01 sec
      Start  3: Cabana_CartesianGrid_test
 3/73 Test  #3: Cabana_CartesianGrid_test ....................   Passed    0.01 sec
      Start  4: Cabana_SoA_test
 4/73 Test  #4: Cabana_SoA_test ..............................   Passed    0.01 sec
      Start  5: Cabana_AoSoA_test_OPENMP_nt_1
 5/73 Test  #5: Cabana_AoSoA_test_OPENMP_nt_1 ................   Passed    0.01 sec
      Start  6: Cabana_AoSoA_test_OPENMP_nt_2
 6/73 Test  #6: Cabana_AoSoA_test_OPENMP_nt_2 ................   Passed    0.01 sec
      Start  7: Cabana_DeepCopy_test_OPENMP_nt_1
 7/73 Test  #7: Cabana_DeepCopy_test_OPENMP_nt_1 .............   Passed    0.03 sec
      Start  8: Cabana_DeepCopy_test_OPENMP_nt_2
 8/73 Test  #8: Cabana_DeepCopy_test_OPENMP_nt_2 .............   Passed    0.03 sec
      Start  9: Cabana_LinkedCellList_test_OPENMP_nt_1
 9/73 Test  #9: Cabana_LinkedCellList_test_OPENMP_nt_1 .......   Passed    0.01 sec
      Start 10: Cabana_LinkedCellList_test_OPENMP_nt_2
10/73 Test #10: Cabana_LinkedCellList_test_OPENMP_nt_2 .......   Passed    0.01 sec
      Start 11: Cabana_NeighborList_test_OPENMP_nt_1
11/73 Test #11: Cabana_NeighborList_test_OPENMP_nt_1 .........   Passed    0.54 sec
      Start 12: Cabana_NeighborList_test_OPENMP_nt_2
12/73 Test #12: Cabana_NeighborList_test_OPENMP_nt_2 .........   Passed    0.28 sec
      Start 13: Cabana_Parallel_test_OPENMP_nt_1
13/73 Test #13: Cabana_Parallel_test_OPENMP_nt_1 .............   Passed    0.01 sec
      Start 14: Cabana_Parallel_test_OPENMP_nt_2
14/73 Test #14: Cabana_Parallel_test_OPENMP_nt_2 .............   Passed    0.01 sec
      Start 15: Cabana_ParameterPack_test_OPENMP_nt_1
15/73 Test #15: Cabana_ParameterPack_test_OPENMP_nt_1 ........   Passed    0.01 sec
      Start 16: Cabana_ParameterPack_test_OPENMP_nt_2
16/73 Test #16: Cabana_ParameterPack_test_OPENMP_nt_2 ........   Passed    0.01 sec
      Start 17: Cabana_Slice_test_OPENMP_nt_1
17/73 Test #17: Cabana_Slice_test_OPENMP_nt_1 ................   Passed    0.01 sec
      Start 18: Cabana_Slice_test_OPENMP_nt_2
18/73 Test #18: Cabana_Slice_test_OPENMP_nt_2 ................   Passed    0.01 sec
      Start 19: Cabana_Sort_test_OPENMP_nt_1
19/73 Test #19: Cabana_Sort_test_OPENMP_nt_1 .................   Passed    0.04 sec
      Start 20: Cabana_Sort_test_OPENMP_nt_2
20/73 Test #20: Cabana_Sort_test_OPENMP_nt_2 .................   Passed    0.04 sec
      Start 21: Cabana_Tuple_test_OPENMP_nt_1
21/73 Test #21: Cabana_Tuple_test_OPENMP_nt_1 ................   Passed    0.01 sec
      Start 22: Cabana_Tuple_test_OPENMP_nt_2
22/73 Test #22: Cabana_Tuple_test_OPENMP_nt_2 ................   Passed    0.01 sec
      Start 23: Cabana_CommunicationPlan_test_OPENMP_np_1
23/73 Test #23: Cabana_CommunicationPlan_test_OPENMP_np_1 ....   Passed    0.37 sec
      Start 24: Cabana_CommunicationPlan_test_OPENMP_np_2
24/73 Test #24: Cabana_CommunicationPlan_test_OPENMP_np_2 ....   Passed    0.32 sec
      Start 25: Cabana_Distributor_test_OPENMP_np_1
25/73 Test #25: Cabana_Distributor_test_OPENMP_np_1 ..........   Passed    0.31 sec
      Start 26: Cabana_Distributor_test_OPENMP_np_2
26/73 Test #26: Cabana_Distributor_test_OPENMP_np_2 ..........   Passed    0.32 sec
      Start 27: Cabana_Halo_test_OPENMP_np_1
27/73 Test #27: Cabana_Halo_test_OPENMP_np_1 .................   Passed    0.31 sec
      Start 28: Cabana_Halo_test_OPENMP_np_2
28/73 Test #28: Cabana_Halo_test_OPENMP_np_2 .................   Passed    0.31 sec
      Start 29: Cajita_GlobalMesh_test_OPENMP_nt_1
29/73 Test #29: Cajita_GlobalMesh_test_OPENMP_nt_1 ...........   Passed    0.01 sec
      Start 30: Cajita_GlobalMesh_test_OPENMP_nt_2
30/73 Test #30: Cajita_GlobalMesh_test_OPENMP_nt_2 ...........   Passed    0.01 sec
      Start 31: Cajita_IndexSpace_test_OPENMP_nt_1
31/73 Test #31: Cajita_IndexSpace_test_OPENMP_nt_1 ...........   Passed    0.01 sec
      Start 32: Cajita_IndexSpace_test_OPENMP_nt_2
32/73 Test #32: Cajita_IndexSpace_test_OPENMP_nt_2 ...........   Passed    0.01 sec
      Start 33: Cajita_SparseIndexSpace_test_OPENMP_nt_1
33/73 Test #33: Cajita_SparseIndexSpace_test_OPENMP_nt_1 .....   Passed    4.99 sec
      Start 34: Cajita_SparseIndexSpace_test_OPENMP_nt_2
34/73 Test #34: Cajita_SparseIndexSpace_test_OPENMP_nt_2 .....   Passed    2.58 sec
      Start 35: Cajita_Splines_test_OPENMP_nt_1
35/73 Test #35: Cajita_Splines_test_OPENMP_nt_1 ..............   Passed    0.01 sec
      Start 36: Cajita_Splines_test_OPENMP_nt_2
36/73 Test #36: Cajita_Splines_test_OPENMP_nt_2 ..............   Passed    0.01 sec
      Start 37: Cajita_GlobalGrid_test_OPENMP_np_1
37/73 Test #37: Cajita_GlobalGrid_test_OPENMP_np_1 ...........   Passed    0.26 sec
      Start 38: Cajita_GlobalGrid_test_OPENMP_np_2
38/73 Test #38: Cajita_GlobalGrid_test_OPENMP_np_2 ...........   Passed    0.31 sec
      Start 39: Cajita_LocalGrid_test_OPENMP_np_1
39/73 Test #39: Cajita_LocalGrid_test_OPENMP_np_1 ............   Passed    0.31 sec
      Start 40: Cajita_LocalGrid_test_OPENMP_np_2
40/73 Test #40: Cajita_LocalGrid_test_OPENMP_np_2 ............   Passed    0.31 sec
      Start 41: Cajita_IndexConversion_test_OPENMP_np_1
41/73 Test #41: Cajita_IndexConversion_test_OPENMP_np_1 ......   Passed    0.92 sec
      Start 42: Cajita_IndexConversion_test_OPENMP_np_2
42/73 Test #42: Cajita_IndexConversion_test_OPENMP_np_2 ......   Passed    0.72 sec
      Start 43: Cajita_LocalMesh3d_test_OPENMP_np_1
43/73 Test #43: Cajita_LocalMesh3d_test_OPENMP_np_1 ..........   Passed    1.73 sec
      Start 44: Cajita_LocalMesh3d_test_OPENMP_np_2
44/73 Test #44: Cajita_LocalMesh3d_test_OPENMP_np_2 ..........   Passed    2.63 sec
      Start 45: Cajita_LocalMesh2d_test_OPENMP_np_1
45/73 Test #45: Cajita_LocalMesh2d_test_OPENMP_np_1 ..........   Passed    0.28 sec
      Start 46: Cajita_LocalMesh2d_test_OPENMP_np_2
46/73 Test #46: Cajita_LocalMesh2d_test_OPENMP_np_2 ..........   Passed    0.34 sec
      Start 47: Cajita_Array3d_test_OPENMP_np_1
47/73 Test #47: Cajita_Array3d_test_OPENMP_np_1 ..............   Passed    0.37 sec
      Start 48: Cajita_Array3d_test_OPENMP_np_2
48/73 Test #48: Cajita_Array3d_test_OPENMP_np_2 ..............   Passed    0.34 sec
      Start 49: Cajita_Array2d_test_OPENMP_np_1
49/73 Test #49: Cajita_Array2d_test_OPENMP_np_1 ..............   Passed    0.31 sec
      Start 50: Cajita_Array2d_test_OPENMP_np_2
50/73 Test #50: Cajita_Array2d_test_OPENMP_np_2 ..............   Passed    0.32 sec
      Start 51: Cajita_Halo3d_test_OPENMP_np_1
51/73 Test #51: Cajita_Halo3d_test_OPENMP_np_1 ...............   Passed    0.98 sec
      Start 52: Cajita_Halo3d_test_OPENMP_np_2
52/73 Test #52: Cajita_Halo3d_test_OPENMP_np_2 ...............   Passed    1.70 sec
      Start 53: Cajita_Halo2d_test_OPENMP_np_1
53/73 Test #53: Cajita_Halo2d_test_OPENMP_np_1 ...............   Passed    0.29 sec
      Start 54: Cajita_Halo2d_test_OPENMP_np_2
54/73 Test #54: Cajita_Halo2d_test_OPENMP_np_2 ...............   Passed    0.34 sec
      Start 55: Cajita_SplineEvaluation3d_test_OPENMP_np_1
55/73 Test #55: Cajita_SplineEvaluation3d_test_OPENMP_np_1 ...   Passed    1.14 sec
      Start 56: Cajita_SplineEvaluation3d_test_OPENMP_np_2
56/73 Test #56: Cajita_SplineEvaluation3d_test_OPENMP_np_2 ...   Passed    0.72 sec
      Start 57: Cajita_SplineEvaluation2d_test_OPENMP_np_1
57/73 Test #57: Cajita_SplineEvaluation2d_test_OPENMP_np_1 ...   Passed    0.32 sec
      Start 58: Cajita_SplineEvaluation2d_test_OPENMP_np_2
58/73 Test #58: Cajita_SplineEvaluation2d_test_OPENMP_np_2 ...   Passed    0.32 sec
      Start 59: Cajita_Interpolation3d_test_OPENMP_np_1
59/73 Test #59: Cajita_Interpolation3d_test_OPENMP_np_1 ......   Passed    0.40 sec
      Start 60: Cajita_Interpolation3d_test_OPENMP_np_2
60/73 Test #60: Cajita_Interpolation3d_test_OPENMP_np_2 ......   Passed    0.40 sec
      Start 61: Cajita_Interpolation2d_test_OPENMP_np_1
61/73 Test #61: Cajita_Interpolation2d_test_OPENMP_np_1 ......   Passed    0.32 sec
      Start 62: Cajita_Interpolation2d_test_OPENMP_np_2
62/73 Test #62: Cajita_Interpolation2d_test_OPENMP_np_2 ......   Passed    0.32 sec
      Start 63: Cajita_BovWriter_test_OPENMP_np_1
63/73 Test #63: Cajita_BovWriter_test_OPENMP_np_1 ............   Passed    0.40 sec
      Start 64: Cajita_BovWriter_test_OPENMP_np_2
64/73 Test #64: Cajita_BovWriter_test_OPENMP_np_2 ............   Passed    0.40 sec
      Start 65: Cajita_Parallel_test_OPENMP_np_1
65/73 Test #65: Cajita_Parallel_test_OPENMP_np_1 .............   Passed    0.41 sec
      Start 66: Cajita_Parallel_test_OPENMP_np_2
66/73 Test #66: Cajita_Parallel_test_OPENMP_np_2 .............   Passed    0.37 sec
      Start 67: Core_tutorial_01
67/73 Test #67: Core_tutorial_01 .............................   Passed    0.01 sec
      Start 68: Core_tutorial_02
68/73 Test #68: Core_tutorial_02 .............................   Passed    0.01 sec
      Start 69: Core_tutorial_03
69/73 Test #69: Core_tutorial_03 .............................   Passed    0.01 sec
      Start 70: Core_tutorial_04_unmanaged
70/73 Test #70: Core_tutorial_04_unmanaged ...................   Passed    0.01 sec
      Start 71: Core_tutorial_05_openmp
71/73 Test #71: Core_tutorial_05_openmp ......................   Passed    0.01 sec
      Start 72: Core_tutorial_10_neighbor
72/73 Test #72: Core_tutorial_10_neighbor ....................   Passed    0.01 sec
      Start 73: Core_tutorial_10_simd
73/73 Test #73: Core_tutorial_10_simd ........................   Passed    0.01 sec
```

File: cajita/unit_test/tstHalo2d.hpp [hunk 1]
```diff
@@ -398,12 +398,16 @@ TEST( TEST_CATEGORY, not_periodic_test )
...
 
+    gatherScatterTest( partitioner, is_dim_periodic );
+
     // Test with different block configurations to make sure all the
     // dimensions get partitioned even at small numbers of ranks.
-    gatherScatterTest( partitioner, is_dim_periodic );
-    std::swap( ranks_per_dim[0], ranks_per_dim[1] );
-    partitioner = ManualBlockPartitioner<2>( ranks_per_dim );
-    gatherScatterTest( partitioner, is_dim_periodic );
+    if ( ranks_per_dim[0] != ranks_per_dim[1] )
```
Comment (sslattery) [inline cajita/unit_test/tstHalo2d.hpp:405]: Good idea - no reason to test duplicate combinations.

**Review by sslattery — APPROVED** [review]
Very nice - this is a good start. The few that are over a second we can look into a bit more in another PR.

**Pull request merged**